threescale_api/resources.py

```diff
@@ -331,6 +331,23 @@ def create_token(self, entity_id: int, params, **kwargs):
         return utils.extract_response(response=response)
 
 
+class PersonalAccesses(DefaultClient):
```

Comment:
Extend class ProviderAccounts instead of creating new one. ProviderAccounts has same endpoint `/api/admin/users` as you need.